integration/ui/src/main/resources/PhenoTips/VariantTableWidget.xml

```diff
@@ -146,12 +147,12 @@
         end : The end position (exclusive),
         referenceBases : The reference bases for this variant,
         alternateBases : The bases that appear instead of the reference bases,
-        info : {
-          EXOMISER_VARIANT_SCORE : The exomiser score for the variant,
-          GENE : The name of the gene to which this variant belongs,
-          EXOMISER_GENE_VARIANT_SCORE : The exomiser harmfulness of the gene,
-          GENE_EFFECT : The effect of this mutation (ie MISSENSE, STOPGAIN etc.),
-          EXAC_AF : The exac allele frequency of this variant
+        zygosity : The exac allele frequency of this variant,
+        effect : The effect of this mutation (ie MISSENSE, STOPGAIN etc.),
+        score : The exomiser score for the variant,
+        annotations : {
+          geneSymbol : The HGNC ID of the gene to which this variant belongs,
```

Comment:
Technically just the gene symbol. The HGNC ID is something else. E.g., http://www.genenames.org/cgi-bin/gene_symbol_report?hgnc_id=HGNC:16974